README.md

```diff
@@ -0,0 +1,36 @@
+# rpmdeplint container image
+
+This repository contains bits needed to build a container image for the [rpmdeplint](https://pagure.io/rpmdeplint). This image can be later used by the Fedora CI.
+
+## Development
+
+For testing purposes, you can either build the image locally or you can open a pull request in this repository, and CI will build the image for you.
+
+To build the image locally, type:
+
+```shell
+podman build -t fedoraci/rpmdeplint:devel .
+```
+
+TODO: we should validate that the final image is not completely broken; however the [runtest.sh](./runtest.sh) script is currently empty :/
+
+### Testing
+
+You can run and test the image locally. Simply run:
+
+```shell
+$ podman run -ti --rm fedoraci/rpmdeplint:devel /rpmdeplint/run_rpmdeplint.py --help
+usage: run_rpmdeplint.py [-h] --task-id TASK_ID --release RELEASE
+
+optional arguments:
+  -h, --help            show this help message and exit
+  --task-id TASK_ID, -t TASK_ID
+                        Koji task id
+  --release RELEASE, -r RELEASE
+                        release, like f31
+
+```
+
+### Note about promoting to production
+
+Merging pull requests to the master branch also triggers build in CI. However, such images are **not*** automatically promoted and used by the CI pipelines. In order to promote a new image to production, you need to explicitly say so in the [rpmdeplint-pipeline](https://github.com/fedora-ci/rpmdeplint-pipeline/README.md#promoting-new-rpmdeplint-image-to-production) repository.
```

Comment:
Small change:
"Merging pull requests to the master branch also triggers a build in CI."

Reply:
Updated, thanks ;)